lib/Madara.lua

```diff
@@ -106,7 +107,13 @@ end
 ---@param url string
 ---@return string
 function defaults:getPassage(url)
-	return table.concat(map(GETDocument(self.expandURL(url)):select("div.text-left p"), text), "\n")
+	local htmlElement = GETDocument(self.expandURL(url)):selectFirst("div.text-left")
+
+	-- Remove/modify unwanted HTML elements to get a clean webpage.
+	htmlElement:removeAttr("style") -- Remove text style
```

Comment:
This line is not needed, see source of `pageOfElem` (https://github.com/shosetsuorg/kotlin-lib/blob/development/src/main/kotlin/app/shosetsu/lib/lua/ShosetsuLuaLib.kt#L188 and https://github.com/shosetsuorg/kotlin-lib/blob/development/src/main/resources/pageOfElem.lua).